src/TurbulenceConvection/update_aux.jl

```diff
@@ -152,8 +152,7 @@ function update_aux!(
         end
     end
 
-    @. aux_en_f.w =
-        wcomponent(CCG.WVector(prog_gm_f.w)) / (1 - Ifb(aux_bulk.area))
+    @. aux_en_f.w = toscalar(prog_gm_f.w) / (1 - Ifb(aux_bulk.area))
```

Comment:
This looks wrong. We should not have `toscalar` anywhere?

Reply:
I think we just need to rebase, I'll rebase now

Reply:
Ah, I messed up the rebase, force-pushing a fix now.